src/map/pc.cpp

```diff
@@ -8617,8 +8617,11 @@ int pc_dead(struct map_session_data *sd,struct block_list *src)
 		else
 			job_penalty = 0;
 
-		if (base_penalty || job_penalty)
-			pc_lostexp(sd, base_penalty, job_penalty);
+		if (base_penalty || job_penalty) {
+			short insurance_idx = pc_search_inventory(sd, ITEMID_NEW_INSURANCE);
+			if (insurance_idx < 0 || pc_delitem(sd, insurance_idx, 1, 0, 1, LOG_TYPE_CONSUME) != 0)
```

Comment:
Should we put ifdef RENEWAL around these two lines?
@aleos89

Reply:
Since we are renewal focused, I would say no. The only time I would wrap it is if it had functionality in pre-renewal and was changed or removed.